### PR TITLE
[MCC-195656] Loader factory can support both old format and v1 format

### PR DESCRIPTION
**Pull request by junshao**
- Added two set of sheet definitions: old format and v1 format.
- Added a factory that can support both formats.
- Incremented NuGet packages versions.
- Added unit tests.
- Added SpecFlow tests, though it cannot be shown in VS 2015 Test Explorer.

@art2003 @chenghuang-mdsol @echen-mdsol Please review and merge if OK. Thanks.

### CODE REVIEW

**Comment by art2003** [comment]
We will need to review all feature files with SQA team

**Comment by junshao** [comment]
This feature file is to integration test the loader APIs, and isn't the feature test owned by SQA.

File: Medidata.Cloud.ExcelLoader/CellTypeValueConverterManager.cs [hunk 1]
```diff
@@ -62,9 +63,18 @@ public object GetCSharpValue(Cell cell)
                 throw new NotSupportedException(msg);
             }
 
-            var propType = Type.GetType(cell.GetMdsolAttribute("type"), true);
+            var propType = GetType(cell.GetMdsolAttribute("type"));
             var propValue = Convert.ChangeType(value, propType);
```
Comment (art2003) [inline Medidata.Cloud.ExcelLoader/CellTypeValueConverterManager.cs]: can propType be Null here, prior ChangeType call

**Comment by art2003** [comment]
@junshao thanks for clarification

File: Medidata.Cloud.ExcelLoader/CellTypeValueConverterManager.cs (hunk 1 above)
Comment (junshao) [inline Medidata.Cloud.ExcelLoader/CellTypeValueConverterManager.cs]: `GetType()` can always return an instance of `Type`. If cannot find the type, `GetType()` will throw exception.

Comment (art2003) [inline Medidata.Cloud.ExcelLoader/CellTypeValueConverterManager.cs]: thanks, merging

**Comment by art2003** [comment]
LGTM

**Pull request merged**